woocommerce/woocommerce-ios
# Show Jetpack banner and benefit modal for non-Jetpack sites

Closes: #9216 


## Description

This PR updates the logic of Jetpack benefit banner to show it when the user logs in to a non-Jetpack site. This helps users know what to expect when using the app without Jetpack.

Since Jetpack setup is still being implemented, the Jetpack banner has been updated to display a CTA to open wp-admin instead.

## Testing instructions

Prerequisite: Make sure that you have access to a self-hosted site with Woo but not Jetpack.
- Disable the feature flag `jetpackSetupWithApplicationPassword` and build the app.
- Log in to your test store with the correct credentials.
- After the login succeeds, notice on the dashboard screen that the Jetpack banner is present.
- Tap the Jetpack banner, the Jetpack benefit modal should be presented. Notice the CTAs should be "Open wp-admin" and "Dismiss".
- Tap "Open wp-admin", a Safari web view should be presented pointing to the Jetpack install page.


## Screenshots


https://user-images.githubusercontent.com/5533851/226545563-a15568b8-0d6d-46cb-9ed5-8570952fbbe7.mp4

| JCP | Jetpack setup enabled for non-JP site | Jetpack setup disabled | 
| ----- | ----- | ----- |
| <img src="https://user-images.githubusercontent.com/5533851/226547203-a2198c31-9ada-4cac-b39d-f9c177620ea1.png" width=320 /> | <img src="https://user-images.githubusercontent.com/5533851/226546624-f4ca328e-1568-435f-ad30-24ef63c747ca.png" width=320 /> | <img src="https://user-images.githubusercontent.com/5533851/226546525-c7f39dfe-464c-4461-bb34-19040fd2db97.png" width=320 /> |


---
